tests/resources/vastForBeaconTest.xml

```diff
@@ -6,33 +6,33 @@
 			<Description/>
 			<Survey/>
 			<Impression id="DART"><![CDATA[
-				http://projects.kaltura.com/beacons/CatchBeacon.php?id=10&eventType=impression
+				http://dev-backend26.dev.kaltura.com/p/1/testing/trackBeacon?id=10&eventType=impression
 			]]></Impression>
 			<Creatives>
 				<Creative sequence="1" AdID="">
 					<Linear>
 						<Duration>00:00:15</Duration>
 						<TrackingEvents>
 							<Tracking event="start"><![CDATA[
-								http://dev-backend3.dev.kaltura.com/p/1/testing/trackBeacon?id=10&eventType=start1
```

Comment:
you can change this address to localhost - as these tests are run on play servers that have the testing manager